flink/v1.18/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergStreamWriterMetrics.java

```diff
@@ -79,7 +80,7 @@ void updateFlushResult(WriteResult result) {
     Arrays.stream(result.deleteFiles())
         .forEach(
             deleteFile -> {
-              deleteFilesSizeHistogram.update(deleteFile.fileSizeInBytes());
+              deleteFilesSizeHistogram.update(ContentFileUtil.contentSizeInBytes(deleteFile));
```

Comment:
@nastra, I cherry-picked all Flink changes across all versions, unlike the prototype PR you reviewed.